### ISSUE TITLE
[Feature] Customizable hotkeys

### ISSUE THREAD

**Issue by CortexReaver** [shayne/PowerDimmer#18]
Subj, interested particularly in activate\deactivate dimming.

**Comment by hkpD** [comment]
Agreed. These hotkeys are really strange. You have to press SHIFT + START + D for one thing, then ALT + START + S for another thing, then CTRL + START + ALT + D for something else (???)

At least turn SHIFT + START + D into ALT + START + D so it's at least consistent with two of the other hotkeys.
